openquake/hazardlib/gsim/kotha_2020.py

```diff
@@ -0,0 +1,672 @@
+    Europe, Bulletin of Earthquake Engineering, under review
+
+    The GMPE is desiged for calibration of the stress parameter term
+    (a multiple of the fault-to-fault variability, tau_f) an attenuation
+    scaling term (c3) and a statistical uncertainty term (sigma_mu). The
+    statistical uncertainty is a scalar factor dependent on period, magnitude
+    and distance. These are read in from hdf5 upon instantiation and
+    interpolated to the necessary values.
+
+    In the core form of the GMPE no site term is included. This will be
+    added in the subclasses.
+
+    :param c3:
+        User supplied table for the coefficient c3 controlling the anelastic
+        attenuation as an instance of :class:
+        `openquake.hazardlib.gsim.base.CoeffsTable`. If absent, the value is
+        taken from the normal coefficients table.
+
+    :param sigma_mu_epsilon:
+        The number by which to multiply the epistemic uncertainty (sigma_mu)
+        for the adjustment of the mean ground motion.
+    """
+    experimental = True
+
+    #: Supported tectonic region type is 'active shallow crust'
+    DEFINED_FOR_TECTONIC_REGION_TYPE = const.TRT.ACTIVE_SHALLOW_CRUST
+
+    #: Set of :mod:`intensity measure types <openquake.hazardlib.imt>`
+    #: this GSIM can calculate. A set should contain classes from module
+    #: :mod:`openquake.hazardlib.imt`.
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([
+        PGA,
+        PGV,
+        SA
+    ])
+
+    #: Supported intensity measure component is the geometric mean of two
+    #: horizontal components
+    DEFINED_FOR_INTENSITY_MEASURE_COMPONENT = const.IMC.RotD50
+
+    #: Supported standard deviation types are inter-event, intra-event
+    #: and total
+    DEFINED_FOR_STANDARD_DEVIATION_TYPES = set([
+        const.StdDev.TOTAL,
+        const.StdDev.INTER_EVENT,
+        const.StdDev.INTRA_EVENT
+    ])
+
+    #: Required site parameter is not set
+    REQUIRES_SITES_PARAMETERS = set()
+
+    #: Required rupture parameters are magnitude and hypocentral depth
+    REQUIRES_RUPTURE_PARAMETERS = set(('mag', 'hypo_depth'))
+
+    #: Required distance measure is Rjb (eq. 1).
+    REQUIRES_DISTANCES = set(('rjb', ))
+
+    def __init__(self, sigma_mu_epsilon=0.0, c3=None, ergodic=True):
+        """
+        Instantiate setting the sigma_mu_epsilon and c3 terms
+        """
+        super().__init__()
+        if isinstance(c3, dict):
+            # Inputing c3 as a dictionary sorted by the string representation
+            # of the IMT
+            c3in = {}
+            for c3key in c3:
+                c3in[from_string(c3key)] = {"c3": c3[c3key]}
+            self.c3 = CoeffsTable(sa_damping=5, table=c3in)
+        else:
+            self.c3 = c3
+
+        self.sigma_mu_epsilon = sigma_mu_epsilon
+        self.ergodic = ergodic
+        if self.sigma_mu_epsilon:
+            # Connect to hdf5 and load tables into memory
+            self.retreive_sigma_mu_data()
+        else:
```

Comment:
Is this an US/British spelling issue, or a bona fide misprint? retreive -> retrieve

Reply:
It's a typo. Now fixed